src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx

```diff
@@ -245,65 +325,210 @@ export class CancerSummaryChart extends React.Component<CancerSummaryChartProps,
         };
     }
 
-    render() {
+    private domainPadding() {
+        return PLOT_DATA_PADDING_PIXELS;
+    }
+
+    private categoryAxisDomainPadding() {
+        return this.domainPadding();
+    }
+
+    private countAxisDomainPadding() {
+        return this.domainPadding();
+    }
+
+    private chartDomainPadding() {
+        return {
+            y: this.countAxisDomainPadding(),
+            x: this.categoryAxisDomainPadding()
+        };
+    }
+
+    private barChartHeight() {
+        return 400;
+    }
 
-        // if we're not showing result broken down the alterations then we need to merge
+    @computed get scatterChartHeight() {
+        return this.scatterChartExtent;
+    }
+
+    @computed get scatterChartExtent() {
```

Comment:
seems poorly named.  If it's only being used by the height getter above, why not just put the calculation there?